sys-process/anacron/anacron-2.3-r4.ebuild

```diff
@@ -12,14 +12,15 @@ SRC_URI="mirror://sourceforge/${PN}/${P}.tar.gz"
 LICENSE="GPL-2+"
 SLOT="0"
 KEYWORDS="amd64 ~arm ppc ~riscv x86"
-IUSE=""
 
 DEPEND="
-	sys-process/cronbase"
+	sys-process/cronbase
```

Comment:
Doesn't need this be in ```BDEPEND``` as the created user (acct-* depends on cronbase) is used during ```src_install```?

Reply:
Hm
I'm not sure, BDEPEND is installed in the system performing the build, and DEPEND is installed in the system being built... Maybe it should be in both (chown will fail if it doesn't know the UID/GID, but I don't know what portage do when crosscompiling)

@arthurzam what do you think?

Reply:
I'm actually unsure what happens in this case. Since we use `acct-group/*` package, the GID should be the same for both host and "cross". If I'm not wrong, we only care for the group name and GID.

Reply:
Ok, I checked and sys-process/cronbase is just a bunch of dirs and a script which is unlikely to run at build time, so what about moving cronbase to RDEPEND (leaving DEPEND empty) and add acct-group/cron to BDEPEND

Reply:
@thesamesam any ideas? (I asked in irc but...)

Reply:
Interesting indeed, maybe DEPEND + BDEPEND is the correct choice. We can of course always update that if someone reports an error, this PR shouldn't really wait for that.